addons/html_editor/static/src/editor/list/list_plugin.js

```diff
@@ -311,6 +314,54 @@ export class ListPlugin extends Plugin {
         }
     }
 
+    // --------------------------------------------------------------------------
+    // Helpers for normalize
+    // --------------------------------------------------------------------------
+
+    preserveSelection() {
```

Comment:
looks like this should be in selection plugin?

Reply:
Probably.
But this is rather incomplete and only handles the use cases of list normalization so far...

Reply:
it's fine to have incomplete implementation imo. and it's better to put code at the proper location, so there is a higher chance that someone finds it (and improve it if necessary) later